packages/repository/test/unit/repository-mixin/repository-mixin.test.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {
+  RepositoryMixin,
+  juggler,
+  DataSourceConstructor,
+  Class,
+} from '../../../';
+import {Application, Component} from '@loopback/core';
+
+// tslint:disable:no-any
+
+describe('RepositoryMixin', () => {
+  const appWithRepoMixin = RepositoryMixin(Application);
+
+  class NoteRepo {
+    model: any;
+
+    constructor() {
+      const ds: juggler.DataSource = new DataSourceConstructor({
+        name: 'db',
+        connector: 'memory',
+      });
+
+      this.model = ds.createModel(
+        'note',
+        {title: 'string', content: 'string'},
+        {},
+      );
+    }
+  }
+
+  class TestComponent {
+    repositories = [NoteRepo];
+  }
+
+  function expectNoteRepoToBeBound(myApp: Application) {
```

Comment:
Please move the helpers to the bottom of the file, see http://loopback.io/doc/en/contrib/style-guide.html#layout-of-test-files

Reply:
Feedback Applied.